js-envs/src/test/scala/org/scalajs/jsenv/test/ComTests.scala

```diff
@@ -188,6 +188,17 @@ trait ComTests extends AsyncTests {
   }
 
   @Test
+  def noMessageTest = {
+    val com = comRunner(s"""
+      for (x = 0; x < 10000; ++x); // hopefully this slows us down
```

Comment:
I wanted to avoid `setTimeout` in order to not rely on it (I'd have to move the test to another trait). Maybe that is the better option